website/server/models/user/schema.js

```diff
@@ -494,7 +494,7 @@ let schema = new Schema({
     mp: {type: Number, default: 10},
     exp: {type: Number, default: 0},
     gp: {type: Number, default: 0},
-    lvl: {type: Number, default: 1},
+    lvl: {type: Number, default: 1, min: 1},
```

Comment:
No need in my opinion. The site behaves normally with negative levels as far as I can tell from a quick test, and even if there were problems, there'd be no need for a migration before merging this (and we'd probably already have heard from any users affected by it).